Makefile

```diff
@@ -129,21 +129,21 @@ tidy:
 # ------------
 
 ## Generate deepcopy code, kubernetes manifests and docs.
-generate:
+generate-all:
```

Comment:
Let's get rid of `generate`/`generate-all` entirely, we'll leverage Boilerplate's generate targets.